### DISCUSSION TITLE
v1.1.3 not working under 10.11.4, needs 10.12!?

### DISCUSSION THREAD

**Discussion by meinefresse** [halo/LinkLiar#48] [Q&A]
hi

when i wanna start v1.1.3 i get an error "needs 10.12" which i also find in the plist.
after change it to 10.11 it doesn't work with the same error.
best of all, it's all the same with 1.1.2, 1.1.1 and so all, all with the same error.....???

any help is appreciated!
or a tip for another mac spoofing app!

thx alot and kind regards

**Comment by halo** [comment]
Hi! Thank you for your report.

Are you building it using Xcode (which branch?) or installing it via homebrew or downloading the executable directly? I assume you use macOS `10.11`?

As for the other spoofing apps, have a look on this page: https://halo.github.io/LinkLiar in the section "Which other apps are out there?"

As a side note, I haven't had a chance to work much on this project. But I'm trying still to get the current issues resolved :)

**Reply by meinefresse** [reply]
hi again!

i'm using the app directly, cause i've got no idea how to build it by my own.
btw, there's a basic instruction missing on github on how to so...

and yes, i'm on 10.11.4
what irritated me, was the error with the needed 10.12 version for all versions as i said.

thx alot for the "which other apps" section!!!

kind regards!

**Comment by halo** [comment]
This is really peculiar as I've been using those versions of LinkLiar on older macOS versions myself. In particular, LinkLiar `1.1.3` (from https://github.com/halo/LinkLiar/releases/tag/1.1.3) should work on macOS `10.11.4` (El Capitan). I've been downloading it myself from Github back then and it worked.

When `1.1.3` was released, it was explicitly configured to be compatible all the way back to macOS `10.8` per this variable: 
https://github.com/halo/LinkLiar/blob/85456307120fcb4dd7d5f510299a53b7994207a0/Link.xcodeproj/project.pbxproj#L483

I don't have any machine with El Capitan any more 🙁 And even if I had one, I think it would work.

>  i get an error "needs 10.12"

Could you specify that a bit more? Maybe a screenshot or so?

> thx alot for the "which other apps" section!!!

You're welcome. Often I'll just use `ifconfig` in the Terminal if I briefly need to change my MAC address because it's very fast, but of course not as convenient on a long-term basis. If you find any other useful GUI tool, let me know so I can add it to the list, thank you.
